workspaces/global-floating-action-button/plugins/global-floating-action-button/README.md

```diff
@@ -10,6 +10,56 @@ This plugin has been added to the example app in this workspace, meaning it can
 
 ### Installation
 
+#### Installing as a dynamic plugin?
+
+The sections below are relevant for static plugins. If the plugin is expected to be installed as a dynamic one:
+
+- Follow https://github.com/redhat-developer/rhdh/blob/main/docs/dynamic-plugins/installing-plugins.md
+- Add content of `app-config.dynamic.yaml` into `app-config.local.yaml`.
+- To configure a plugin as a Floating Action Button (FAB), you need to specify the `global.floatingactionbutton/component` mount point in your plugin configuration, as shown below using the bulk-import plugin as an example:
+
+  ```yaml
+  dynamicPlugins:
+    frontend:
+      red-hat-developer-hub.backstage-plugin-bulk-import:
+        # start of fab config
+        mountPoints:
+          - mountPoint: global.floatingactionbutton/component
+            config:
+              slot: 'page-end'
+              icon: bulkImportIcon
+              label: 'Bulk import'
+              toolTip: 'Register multiple repositories in bulk'
+              to: /bulk-import/repositories
+        # end of fab config
+        appIcons:
+          - name: bulkImportIcon
+            importName: BulkImportIcon
+        dynamicRoutes:
+          - path: /bulk-import/repositories
+            importName: BulkImportPage
+            menuItem:
+              icon: bulkImportIcon
+              text: Bulk import
+  ```
+
+- To configure a Floating Action Button (FAB) that opens an external link, specify the `global.floatingactionbutton/component` mount point in the `backstage-plugin-global-floating-action-button` plugin, as shown below:
+
+  ```yaml
+  dynamicPlugins:
+    frontend:
+      red-hat-developer-hub.backstage-plugin-global-floating-action-button:
+        mountPoints:
+          - mountPoint: application/provider
+            importName: DynamicGlobalFloatingActionButton
+          - mountPoint: global.floatingactionbutton/component
+            config:
```

Comment:
I know that there is no need for a component, but maybe we should just export a NullComponent that does nothing so that we don't need to "hack" the mount point loading mechanism? Myself is unsure on that... Let us discuss this in a call.